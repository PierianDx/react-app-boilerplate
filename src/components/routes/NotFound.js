import React from 'react';

const NotFound = () => (
  <section className="section">
    <h1>404 Not Found</h1>;
  </section>
);

export default NotFound;
