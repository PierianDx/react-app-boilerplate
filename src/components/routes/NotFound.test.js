import React from 'react';
import { shallow } from 'enzyme';

import NotFound from './NotFound';

describe('NotFound', () => {
  it('should render correctly', () => {
    const comp = shallow(<NotFound />);
    expect(comp).toMatchSnapshot();
  });
});
