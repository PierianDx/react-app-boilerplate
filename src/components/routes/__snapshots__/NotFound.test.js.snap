// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should render correctly 1`] = `
<section
  className="section"
>
  <h1>
    404 Not Found
  </h1>
</section>
`;
